orcid-web/src/main/java/org/orcid/frontend/web/controllers/ShibbolethController.java

```diff
@@ -159,7 +159,7 @@ public ModelAndView signinHandler(HttpServletRequest request, HttpServletRespons
             
             ProfileEntity profile = profileEntityCacheManager.retrieve(userConnectionEntity.getOrcid());
             if (profile.getUsing2FA()) {
-                return new ModelAndView("redirect:" + orcidUrlManager.getBaseUrl() + "/2fa-signin");
+                return new ModelAndView("institutional_2FA");
```

Comment:
@auumgn this code will redirect to the old 2FA page https://github.com/ORCID/ORCID-Source/pull/5999/files. I don't think that's the expected behavior. Could you give it another look, please